example/udev/00_persistent_netdev

```diff
@@ -0,0 +1,4 @@
+# Persistent netdev example using PCI
```

Comment:
why this is needed?

Reply:
its an example of how to do persistent dev name,
you need the same netdev name across setups for rdma device plugin and macvlan cni

i can add a comment explaining that in the file

Reply:
why the operator is not doing it by default?

Reply:
This was not in the scope, we should discuss more.
At machine boot you need to have some sort of constant that is the same across machines, would that be the PCI address ?